packages/ui/src/hooks/useStore.ts

```diff
@@ -158,8 +172,10 @@ export const useStore = (): Store => {
     state,
     actions: {
       promoteJob,
-      retryJob,
-      retryAll,
+      retryFailedJob,
```

Comment:
Instead of forking by type of the status, can we pass the status as a parameter?
This means that there will be 2 retry functions which will get `queueName` & `status` as parameters, and pass it to the proper api, it can be query-string.
It can eliminate the need for a separate api end point.

Reply:
done